api/envoy/config/core/v3/deprecaetd_example.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package envoy.config.core.v3;
+
+import "envoy/annotations/deprecation.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+
+option java_package = "io.envoyproxy.envoy.config.core.v3";
+option java_outer_classname = "DeprecaetdExampleProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Test message]
+
+// EnumExample
+enum EnumExample {
+  // Enum value deprecated in minor version 5.
+  VAL1 = 0 [deprecated = true, (envoy.annotations.minor_ver_enum) = 5];
```

Comment:
This is technically fully expressive, but might be clearer if it looked like "3.5" to the reader. Not sure if this is actionable.

Reply:
Done.